calcom/cal.com
# #5613 Optimise social images by utilising next image api.

## What does this PR do?

Optimises our OG Images by leveraging Next's Image API. This reduces the image size from 500KB to 80KB 🤯

Fixes #5613

**Environment**: Staging(main branch)

## Type of change

## How should this be tested?

- [ ] Verify that the OG images are still generated properly by checking the OG Image meta tag and visiting that url on the following pages: Meeting booking page (pro/30min), App detail page, any other page on app.cal.com (should show a generic image).

The OG meta tag looks like this: 
![CleanShot 2022-12-13 at 11 15 58@2x](https://user-images.githubusercontent.com/2969573/207290837-cf5272b1-772e-4d56-8ec1-34933b96a46a.jpg)

To verify that the link is working you can double click that "content" attribute and copy+paste that url in a new tab. The image should show up 🔥
